integration/spark2/src/main/scala/org/apache/spark/sql/CarbonDataFrameWriter.scala

```diff
@@ -42,6 +42,10 @@ class CarbonDataFrameWriter(sqlContext: SQLContext, val dataFrame: DataFrame) {
 
   private def writeToCarbonFile(parameters: Map[String, String] = Map()): Unit = {
     val options = new CarbonOption(parameters)
+    if (options.tempCSV) {
```

Comment:
please add a comment in CarbonOptions.tempCSV to describe it is deprecated. And remove CarbonOptions.compress

Reply:
OK.
move this deprecate information to `CarbonOptions`